source/api_reference/cancel_payment_reference/index.html.md.erb

```diff
@@ -0,0 +1,46 @@
+---
+title: Cancel a payment API reference
+last_reviewed_on: 2022-05-10
+review_in: 6 months
+weight: 34
+---
+
+# Cancel a payment API reference
```

Comment:
I agree but unfortunately, I think it's a limit of the tech docs template. I've asked in the template's Slack channel to see if it's possible, though.

Reply:
I think that skipping heading levels is bad for accessibility but I can check. If not, this could be a solution.

Reply:
yeah could be an issue

Reply:
I would prefer to keep them as headings, yeah, because we'll be linking to specific parts of the pages from elsewhere in the docs. I know we could also achieve this anchors but I don't want the pages to get too messy.

I think the tech docs template has a backlog for improvements, so I can always raise this with them. 

It's messy at the moment but I think it's probably acceptable for now?